<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPipeSettings</class>
 <widget class="QMainWindow" name="MediaPipeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaPipe Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="warning_lbl">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>All settings are applied only after restarting the FoxyFace software!</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="try_gpu_cb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>Try using GPU acceleration</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mfdc_lbl">
      <property name="toolTip">
       <string>The minimum confidence score for the face detection to be considered successful.</string>
      </property>
      <property name="text">
       <string>Min face detection confidence:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="mfdc_sp">
      <property name="toolTip">
       <string>The minimum confidence score for the face detection to be considered successful.</string>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mfpc_lbl">
      <property name="toolTip">
       <string>The minimum confidence score of face presence score in the face landmark detection.</string>
      </property>
      <property name="text">
       <string>Min face presence confidence:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="mfpc_sp">
      <property name="toolTip">
       <string>The minimum confidence score of face presence score in the face landmark detection.</string>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mtc_lbl">
      <property name="toolTip">
       <string>The minimum confidence score for the face tracking to be considered successful.</string>
      </property>
      <property name="text">
       <string>Min tracking confidence:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="mtc_sp">
      <property name="toolTip">
       <string>The minimum confidence score for the face tracking to be considered successful.</string>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
